### ISSUE TITLE
run exec allocator: chrome failed to start:\nmkdir: cannot create directory '//.local': Permission denied

### ISSUE THREAD

**Issue by bertenbobje** [gotenberg/gotenberg#966]
We use Gotenberg running in a container to create PDFs, but after a restart one day ago we get the following error when trying to create a PDF:

`
{"level":"error","ts":1725879898.2939854,"logger":"api","msg":"convert HTML to PDF: convert to PDF: process first start: start process: run exec allocator: chrome failed to start:\nmkdir: cannot create directory '//.local': Permission denied\ntouch: cannot touch '//.local/share/applications/mimeapps.list': No such file or directory\nchrome_crashpad_handler: --database is required\nTry 'chrome_crashpad_handler --help' for more information.\n[23:23:0909/110458.283576:ERROR:socket.cc(120)] recvmsg: Connection reset by peer (104)\n","trace":"f53ce4ba-c459-42cd-8221-b703d79c631b","remote_ip":"10.244.1.18","host":"<POD>","uri":"/forms/chromium/convert/html","method":"POST","path":"/forms/chromium/convert/html","referer":"","user_agent":"Faraday v1.10.3","status":500,"latency":187879781,"latency_human":"187.879781ms","bytes_in":16292,"bytes_out":21}
`

When we run the container as `privileged` with user `root` it works without a problem, but this poses security risks so we would like to avoid this.

**Comment by gulien** [comment]
Hello @bertenbobje 👋 

Looks like it's the same issue than https://github.com/gotenberg/gotenberg/issues/949.

```
securityContext:
  readOnlyRootFilesystem: false
  allowPrivilegeEscalation: false
  privileged: false
  runAsUser: 1001
```

**Comment by bertenbobje** [comment]
Thanks for the reply!

I've tried the same fix as in #949, but for me `readOnlyRootFileSystem` was already set to `false`.

If it is set like below, I get an error:
```
securityContext:
  privileged: false
  runAsUser: 1000
  runAsGroup: 1000
  runAsNonRoot: false
  readOnlyRootFilesystem: false
  allowPrivilegeEscalation: true
```

If it is set like below, it works (as mentioned in the original post)
```
securityContext:
  privileged: true
  runAsUser: 0
  runAsGroup: 1000
  runAsNonRoot: false
  readOnlyRootFilesystem: false
  allowPrivilegeEscalation: true
```

**Comment by bertenbobje** [comment]
That was the issue! I have no idea why it used to work with user PID `1000` but here we are.

```
gotenberg@<pod>:~$ id -nu 1000
id: '1000': no such user
gotenberg@<pod>:~$ id -nu 1001
gotenberg
```

Changing it to `runAsUser: 1001` and `runAsGroup: 1001` fixed it, thanks!
